cw_core/lib/monero_wallet_utils.dart

```diff
@@ -79,6 +79,7 @@ Future<bool> backupWalletFilesExists(String name) async {
       backupAddressListFile.existsSync();
 }
 
+// WARNING: Transaction keys and your Polyseed CANNOT be recovered if this file is deleted
 Future<void> removeCache(String name) async {
```

Comment:
I somehow think that removeCache() should remove cache, as it is (iirc) being called when wallet doesn't want to open.... so imo it still would be better to leave the wallet without polyseed / passphrase / seed offset but in a usable state..